evals/elsuite/basic/includes.py

```diff
@@ -13,19 +13,24 @@ def __init__(
         self,
         completion_fns: list[CompletionFn],
         samples_jsonl: str,
+        ignore_case: bool = False,
         *args,
         **kwargs,
     ):
         super().__init__(completion_fns, *args, **kwargs)
         assert len(completion_fns) == 1, "Includes only supports one completion fn"
         self.samples_jsonl = samples_jsonl
+        self.ignore_case = ignore_case
 
     def eval_sample(self, sample: Any, *_):
         prompt = sample["input"]
         result = self.completion_fn(
             prompt=prompt,
         )
         sampled = result.get_completions()[0]
+        if self.ignore_case:
+            sampled = sampled.lower()
+            sample["ideal"] = [ref.lower() for ref in sample["ideal"]]
```

Comment:
We shouldn't modify `sampled` or `sample` since it would be misleading when you look at the results later. Could you instead compute `includes_answer` below using something like `utils.get_answer(sampled, ref, ignore_case=True)`?

Reply:
Good idea, I'll update 👍